pkg/server/service/loadbalancer/wrr/wrr.go

```diff
@@ -162,12 +174,17 @@ func (b *Balancer) nextServer() (*namedHandler, error) {
 		handler.deadline += 1 / handler.weight
 
 		heap.Push(b, handler)
-		if _, ok := b.status[handler.name]; ok {
-			if _, ok := b.fenced[handler.name]; !ok {
-				// do not select a fenced handler.
-				break
-			}
+		if _, down := b.status[handler.name]; !down {
```

Comment:
Unless I missed something, `b.status` indicates whether a server is up. Maybe renaming `down` to something like up would make the logic clearer?